docs/features/accessibility.md

```diff
@@ -163,6 +163,47 @@ These keyboard shortcuts allow for quick access to content editing features.
 	</tbody>
 </table>
 
+#### Keystrokes for interacting with annotation threads (e.g. comments, track changes suggestions)
+
+<table>
+	<thead>
+		<tr>
+			<th>Action</th>
+			<th>PC</th>
+			<th>Mac</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td>Move focus to the thread when the selection is anchored in its marker</td>
+			<td><kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>E</kbd></td>
+			<td><kbd>⌘⇧E</kbd></td>
+		</tr>
+		<tr>
+			<td>Exit the annotation and move focus back to the edited content</td>
+			<td colspan="2"><kbd>Esc</kbd></td>
+		</tr>
+		<tr>
+			<td>Browse the focused annotation thread or thread comment</td>
+			<td colspan="2"><kbd>Enter</kbd></td>
+		</tr>
+		<tr>
+			<td>Move across internals of the annotation thread</td>
+			<td><kbd>⇥</kbd>, <kbd>Shift</kbd>+<kbd>⇥</kbd></td>
+			<td><kbd>⇥</kbd>, <kbd>⇧⇥</kbd></td>
+		</tr>
+		<tr>
+			<td>Submit the reply while writing a comment</td>
+			<td><kbd>Ctrl</kbd>+<kbd>Enter</kbd></td>
+			<td><kbd>⌘Enter</kbd></td>
+		</tr>
+		<tr>
```

Comment:
Move after "Move focus to the thread when the selection is anchored in its marker" or after "Exit the annotation and move focus back to the edited content" so it has more logical order. Also, I wonder if we shouldn't add in the description somethine like "(when annotation is focused)". Or do you think it is obvious?

Reply:
This is automatically generated by a script to match the a11y help dialog 1:1. If we start messing with it it may become hard to manage. We discussed ordering in the a11y help dialog in the past, this is probably the best way but it would take some effort.